wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -361,6 +362,54 @@
                 }
               }
             }
+          },
+          "entrypoints": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "deployment": {
+                  "type": "string",
+                  "description": "The name of the deployment (either the concrete name or the generateName from the metadata). If not specified, then all deployments are considered."
+                },
+                "pod": {
+                  "type": "string",
+                  "description": "The name of the pod (either the concrete name or the generateName from the metadata) containing the container to apply the entrypoint to. Optional if there is just one pod in the selected deployment (or as a top level entry in the list)."
+                },
+                "container": {
+                  "type": "string",
+                  "description": "The name of the container to apply the entrypoint to. Optional if there is just a single container in the selected pods."
```

Comment:
Is it still actual
```
Optional if there is just a single container in the selected pods."
```
?
Won't entrypoint be applied to all containers in pod if container is missing?

Reply:
True, this is a little bit misleading